Assets/Scripts/Camera/Camera.cs

```diff
@@ -0,0 +1,67 @@
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class Camera : MonoBehaviour
+    {
+        public float sensX;
+        public float sensY;
+        public float cameraChangeY;
+        public float cameraChangeZ;
+        public KeyCode cameraChangeKey;
+        private float _xRotation;
+        private float _yRotation;
+        private bool _isFirstPerson = true;
+        public Transform playerObject;
+        public Transform cameraHolder;
```

Comment:
keep public fields together, then put all private fields. 

```suggestion
        public float sensX;
        public float sensY;
        public float cameraChangeY;
        public float cameraChangeZ;
        public KeyCode cameraChangeKey;
        public Transform playerObject;
        public Transform cameraHolder;
        private float _xRotation;
        private float _yRotation;
        private bool _isFirstPerson = true;
```